omod/src/main/webapp/resources/htmlforms/hiv/hivGreenCard.html

```diff
@@ -4788,27 +4839,75 @@ <h2 class="StepTitle">Patient management</h2>
-                                     answerConceptIds="164942"
-                                     style="dropdown"
-                                     answerLabels="Standard Care"
-                                />
-                                <span style="color:red;"><strong>*</strong></span>
-                            </td>
+                        <tr>
+                           <td>
+                              <table>
+                                <tr id="tbl-fast-track">
+                                    <td>Differentiated Care</td>
+                                    <td>
+                                        <obs conceptId="164947" id="fast-track"
+                                             answerConceptIds="164942,164943,164944,164945,164946"
+                                             style="dropdown"
+                                             answerLabels="Standard Care,Fast Track,Community ART Distribution - HCW Led,Community ART Distribution - Peer Led,Facility ART Distribution Group"
+                                        />
+                                        <span style="color:red;"><strong>*</strong></span>
+                                    </td>
+                                </tr>
+                                    <tr id="tbl-standard">
+                                        <td>Differentiated Care</td>
+                                        <td>
+                                            <obs conceptId="164947" id="standard"
+                                                 answerConceptIds="164942"
+                                                 style="dropdown"
+                                                 answerLabels="Standard Care"
+                                            />
+                                            <span style="color:red;"><strong>*</strong></span>
+                                        </td>
+                                    </tr>
+                              </table>
+                           </td>
+                          <td>
+                              <table>
+                                  <tr>
+                                      <td>&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;
+                                          Medical insurance cover:
+                                          <obs conceptId="159356AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                                               id="insuranceType"
+                                               answerConceptIds="1917AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
+                                                                 1107AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
+                                                                 5622AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                                               style="dropdown"
+                                               answerLabels="National Health Insurance Fund Member(NHIF),None,Other"
```

Comment:
Should we rename this to NHIF? I don't think we need to have the 'member' in the label but we can have a discussion

Reply:
Yes seems logical.